pages/common/dd.md

```diff
@@ -19,6 +19,14 @@
 
 `dd if=/dev/zero of={{file_1GB}} bs=1024 count=1000000`
 
+- Generate a system backup into a img file and show the progress:
+
+`dd if=/dev/{{drive_device}} of={{/path/to/file.img}} status=progress`
+
+- Restore a drive from an img file and show the progress:
```

Comment:
```suggestion
- Restore a drive from an IMG file and show the progress:
```

Reply:
@CleanMachine1 what about this one?

Reply:
I think the `IMG` looks more generic than `img`.